docs/docs/css-modules.md

```diff
@@ -1,28 +1,65 @@
 ---
-title: CSS Modules
+title: Component-Scoped Styles with CSS Modules
 ---
 
-## CSS modules and Gatsby
+Component-scoped CSS allows you to write traditional, portable CSS with minimal side-effects: gone are the worries of selector name collisions or affecting other components' styles.
 
-Gatsby works out of the box with CSS Modules. Here is an [example site that uses CSS modules](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-css-modules).
+Gatsby works out of the box with [CSS Modules](https://github.com/css-modules/css-modules), a popular solution for writing component-scoped CSS. Here is an [example site that uses CSS Modules](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-css-modules).
 
-## What is a CSS module?
+## What is a CSS Module?
 
-Quoting from
-[the CSS Module homepage](https://github.com/css-modules/css-modules):
+Quoting from [the CSS Module homepage](https://github.com/css-modules/css-modules):
 
-> A **CSS Module** is a CSS file in which all class names and animation names
-> are scoped locally by default.
+> A **CSS Module** is a CSS file in which all class names and animation names are scoped locally by default.
 
-CSS Modules are very popular as they let you write CSS like normal but with a lot
-more safety. The tool automatically makes class and animation names unique so
-you don't have to worry about selector name collisions.
+CSS Modules let you write styles in CSS files but consume them as JavaScript objects for additional processing and safety. CSS Modules are very popular because they automatically make class and animation names unique so you don't have to worry about selector name collisions.
 
-## When to use CSS modules
+### CSS Module example
 
```

Comment:
I'm not particularly familiar with CSS Modules and since this example looks the same as regular css maybe it's worth pointing out that the only difference is the file extension?

```suggestion
The CSS in a CSS module is no different than normal CSS, but the extension of the file is different to mark that the file will be processed.
```